android/src/main/java/com/hoxfon/react/RNTwilioVoice/Constants.java

```diff
@@ -26,6 +26,8 @@
     public static final String ACTION_FCM_TOKEN = "ACTION_FCM_TOKEN";
     public static final String ACTION_CLEAR_MISSED_CALLS_COUNT = "CLEAR_MISSED_CALLS_COUNT";
     public static final String ACTION_OPEN_CALL_IN_PROGRESS = "CALL_IN_PROGRESS";
+    public static final String ACTION_ANSWER = "ACTION_ANSWER";
```

Comment:
For not being confused with `ACTION_ACCEPT` I think these constants should be called something similar to `ACTION_JS_ACCEPT` and `ACTION_JS_REJECT`. Or REACT instead of JS. Or FORWARD.

Reply:
done